src/connection.ts

```diff
@@ -175,6 +175,7 @@ export interface InternalConnectionOptions {
   requestTimeout: number;
   rowCollectionOnDone: boolean;
   rowCollectionOnRequestCompletion: boolean;
+  serverSupportsColumnEncryption: boolean;
```

Comment:
Is this connection level option required? 🤔 I see there is another flag returned with the column metadata flags that should be enough to distinguish whether encrypted data is send or not.

Reply:
That flag is returned on a per column basis (is this an encrypted column or not).

This option is at connection level because we need to know to parse the encryption keys (for the result set as a whole) before we parse the metadata for each column. If the server has negotiated the always encrypted feature to be enabled when the connection initializes then this flag should be set to true on the connection.